app/controllers/groups_controller.rb

```diff
@@ -67,6 +72,36 @@ def create
     end
   end
 
+  # POST /groups/:group_id/subgroups
+  def create_subgroup
+    @group = Group.new(group_params)
+
+    respond_to do |format|
+      if @group.save
+        @group.update(location: GroupAddress.create(location_params[:group_address]))
+        Affiliation.create(affiliated: @group, group: Group.find(params[:group_id]))
+        @signup_form = SignupForm.create!(
```

Comment:
interesting use of the custom form! not its intended use, but very aptly deployed here!

as for "intended use," the original purpose of the custom forms was to allow groups to create signup forms for members with an arbitrary set of fields depending on their needs or the context of the signup campaign they're running. (maybe one group would want multiple signup forms, each with different fields, depending on where they're gathering signups.) hence the high degree of abstraction.

for this story, the idea was to  gather the organizer info in the form for the subgroup creation itself, then call `SignupForm.for` here to create the group's landing page with a signup form for new members, and then redirect to the landing page. that functionality is needed for the demo, but currently missing (see comment to matt below sothat he can fake it.) 

note: if you're confused by my calling a singup form a "landing page," note that the `description` page for a signup from contains a bunch of html. if you GET `groups/:group_id/signup_forms/:signup_form_id/signups/new` you will see that the html takes up 2/3 of the page. (a la screenshots here: https://github.com/affinityworks/main/pull/497, https://user-images.githubusercontent.com/6032844/35997242-c5b64728-0ccd-11e8-9d3d-32b338018b4e.png)

IMHO, none of these requirements were well-captured, since they shifted over our crazy weekend before you came on.

all that said,  maybe let's keep something like your use of the SignupForm constructor above? 

we could create two factories on `SignupForm`: `SignupForm.for_organzer` (what you've done above, plus name fields) and rename the current `SignupForm.for` to `SignupForm.for_member`. then call them both here so that we have a signup form for both organizers and members. and (if we want) redirect the organizer to the organizer signup page. although, IMHO, i still think it would be smoother UX to collect all the info we need for the organizer to create a group (including info about the group and their info) in the subgroup creation form.

let's check in with @Matt-Holland and then pair on polishing this PR a bit before merging?